doc/introduction/circuits.rst

```diff
@@ -165,3 +165,82 @@ For example:
         return qml.expval(qml.PauliZ(1))
 
     result = circuit(0.543)
+
+
+Quantum circuits from other frameworks
+--------------------------------------
+
+PennyLane supports creating customized PennyLane templates imported from other frameworks.
+By loading your existing quantum code as a PennyLane template, you add the ability to perform
+analytic differentiation, and interface with machine learning libraries such as PyTorch and TensorFlow.
+Currently, circuits from
+Qiskit, QASM strings or QASM files can be loaded. For this, the PennyLane-Qiskit
+plugin needs to be installed.
+
+Objects for quantum circuits can be loaded outside or directly inside of a
+:class:`~.pennylane.QNode`. Circuits that contain unbound parameters are also
+supported. Parameter binding may happen by passing a dictionary containing the
+parameter-value pairs.
+
+Once a PennyLane template has been created from such a quantum circuit, it can
+be used similarly to other :doc:`templates <templates>` in PennyLane. One important thing to note
+is that custom templates (similar to pre-defined templates) must always be executed
+within a :class:`~.pennylane.QNode`.
+
+.. note::
+    Certain instructions that are specific to the external frameworks might be
+    ignored when loading an external quantum circuit. Warning messages will
+    be emitted for ignored instructions.
+
+The following is an example of loading and calling a parametrized Qiskit ``QuantumCircuit`` object
+while using the :class:`~.pennylane.QNode` decorator:
+
+.. code-block:: python
+
+    from qiskit import QuantumCircuit
+    from qiskit.circuit import Parameter
+    import numpy as np
+
+    dev = qml.device('default.qubit', wires=2)
+
+    theta = Parameter('θ')
+
+    qc = QuantumCircuit(2)
+    qc.rz(theta, [0])
+    qc.rx(theta, [0])
+    qc.cx(0, 1)
+
+    @qml.qnode(dev)
+    def quantum_circuit_with_loaded_subcircuit(x):
+        qml.load(qc, format='qiskit')({theta: x})
```

Comment:
In the user-facing docs, I'd like to focus on specific loaders, i.e. `from_qiskit` etc.
This means two things:
- Explicitly List somewhere exactly which loaders are available (I didn't check compiled docs to see if it showed up in autosummary)
- in the examples, don't use `load`, but rather use the format-specific loaders

Reply:
* I agree, added the autosummary to the ``circuits.rst``
* Modified the examples accordingly